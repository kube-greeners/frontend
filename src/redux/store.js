import { configureStore } from '@reduxjs/toolkit'
import dashboardReducer from './dashboardSlice'
import { apiSlice } from './apiSlice'

export default configureStore({
  reducer: {
    dashboard: dashboardReducer,
    [apiSlice.reducerPath]:apiSlice.reducer
  },
  middleware: getDefaultMiddleware =>
  getDefaultMiddleware().concat(apiSlice.middleware)
})