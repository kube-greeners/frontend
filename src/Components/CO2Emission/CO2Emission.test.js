import Co2Emission from "./CO2Emission";
import renderer from 'react-test-renderer'
import store from "../../redux/store";
import { Provider } from "react-redux";

test("matches snapshot", () => {
  const tree = renderer.create(<Provider store={store}><Co2Emission/></Provider>).toJSON();
  expect(tree).toMatchSnapshot();
});
