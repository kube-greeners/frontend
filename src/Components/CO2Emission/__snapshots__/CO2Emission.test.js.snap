// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": 500,
      "justifyContent": "center",
    }
  }
>
  <div
    className="lds"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
